test/nightly/deploy.test.ts

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import { WebSiteManagementModels } from 'azure-arm-website';
+import { IHookCallbackContext, ISuiteCallbackContext } from 'mocha';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { DialogResponses, getRandomHexString, requestUtils } from '../../extension.bundle';
+import { longRunningTestsEnabled, testUserInput } from '../global.test';
+import { resourceGroupsToDelete, webSiteClient } from './global.resource.test';
+
+suite('Create Web App and deploy', async function (this: ISuiteCallbackContext): Promise<void> {
+    this.timeout(20 * 6 * 1000);
```

Comment:
Let's increase the timeout for this particular test.  I've run it twice, and once Linux timed out and the other time, Mac OS timed out with:

`Error: Timeout of 120000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.`

I'm confident that the test is fine, so it probably just needs like 200000 ms.